keel-web/src/main/kotlin/com/netflix/spinnaker/keel/apidocs/GenericWildcardTypeModelConverter.kt

```diff
@@ -22,12 +21,12 @@ import org.springframework.stereotype.Component
  *
  * @see https://github.com/springdoc/springdoc-openapi/issues/434#issuecomment-586700760
  */
-@Component
+// @Component
```

Comment:
Was this intentional?

Reply:
Yeah, trying to see if we actually need it. I'm going to do some inspection of the API docs & ensure they look correct then either put this back or delete the whole class.

Reply:
I diffed the API doc generated with and without this component and there is no meaningful difference